client/src/common/components/Members/index.jsx

```diff
@@ -64,25 +76,26 @@ class MembersBox extends PureComponent {
               </button>
             )}
           </li>
-          {_map(users, user => (
+          {_map(members, member => (
```

Comment:
I know that was mine code, but could you please seprate this to renderMembersList() method??
woudl much more cleaner